### PR TITLE
Update CHANGELOG.md with ACS Calling SDK changes

### PR DESCRIPTION
**Pull request by kagbakpem**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +21,21 @@
 - `ChatThreadClient` `remove(participant)` method now accepts `CommunicationIdentifier` instead of a string
 - For `Participant` renamed `user` property to `id`
 
+**Azure Communication Calling**
+- `Renderer` renamed to `VideoStreamRenderer`.
```
Comment (JoshuaLai) [inline CHANGELOG.md:25]: Should this be considered a breaking change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,7 +21,21 @@
...
+- `AudioDeviceInfo` removed from `DeviceManager`, please use iOS system API's in your application to switch between audio devices.
```
Comment (JoshuaLai) [inline CHANGELOG.md:26]: Should this be considered a breaking change?

**Review by JoshuaLai — CHANGES_REQUESTED** [review]
Looks good just to tiny questions

File: CHANGELOG.md (hunk 1 above)
Comment (kagbakpem) [reply CHANGELOG.md:25]: Yes, as it requires code change from the end user perspective to keep using the SDK?

Comment (JoshuaLai) [reply CHANGELOG.md:25]: Ah sorry I misinterpreted looked like it was in the wrong section my mistake!

**Review by JoshuaLai — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (kagbakpem) [reply CHANGELOG.md:26]: Yes, as this drop will break previous contract. Code change is needed on User end in order to consume this if the previous API was used.

**Review by tjprescott — APPROVED** [review]

**Pull request merged**